.github/workflows/web-test-e2e.yml

```diff
@@ -23,25 +26,45 @@ jobs:
         with:
           filters: |
             admin:
-              - ".github/workflows/web-test.yml"
+              - ".github/workflows/web-test-e2e.yml"
               - "admin/**"
               - "cli/**"
               - "runtime/**"
               - "web-admin/**"
             auth:
-              - ".github/workflows/web-test.yml"
+              - ".github/workflows/web-test-e2e.yml"
               - "web-auth/**"
             local:
-              - ".github/workflows/web-test.yml"
+              - ".github/workflows/web-test-e2e.yml"
               - "cli/**"
               - "runtime/**"
               - "web-local/**"
             common:
-              - ".github/workflows/web-test.yml"
+              - ".github/workflows/web-test-e2e.yml"
               - "cli/**"
               - "runtime/**"
               - "web-common/**"
 
+      - name: Evaluate e2e tests matrix
+        id: matrix
+        run: |
+          echo "matrix=$(jq -n '[
+              "web-admin" | select(env.FILTER_ADMIN == "true" or env.FILTER_COMMON == "true"),
+              "web-local" | select(env.FILTER_LOCAL == "true" or env.FILTER_COMMON == "true")
+            ]' | jq -c)" >> "$GITHUB_OUTPUT"
+        env:
+          FILTER_ADMIN: ${{ steps.filter.outputs.admin }}
+          FILTER_COMMON: ${{ steps.filter.outputs.common }}
+          FILTER_LOCAL: ${{ steps.filter.outputs.local }}
```

Comment:
It feels like these environment variables are adding an extra layer of indirection. Could we use `filter.outputs` directly in the `jq` command like: `select(${{ steps.filter.outputs.admin }} == "true" or ${{ steps.filter.outputs.common }} == "true")`?

Reply:
Further, can we treat the filter output like a typical boolean, not a string, like: 
`select(${{ steps.filter.outputs.admin }} or ${{ steps.filter.outputs.common }})`
or even without the `select` function:
`if ${{ steps.filter.outputs.admin }} or ${{ steps.filter.outputs.common }} then "web-admin" else empty end`

Reply:
yeah, it doesn't have access to `steps.filter.outputs`.

Reply:
Are you sure? [This example](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#example-usage-of-the-steps-context) seems to show it's possible.